RealmSwift/LinkingObjects.swift

```diff
@@ -469,7 +509,7 @@ public class LinkingObjectsBase: NSObject, NSFastEnumeration {
     internal let objectClassName: String
     internal let propertyName: String
 
-    private var cachedRLMResults: RLMResults?
+    fileprivate var cachedRLMResults: RLMResults?
```

Comment:
Not exactly. `private` is new in Swift 3, and Swift 2's `private` was renamed `fileprivate`.